.pre-commit-config.yaml

```diff
@@ -36,3 +36,10 @@ repos:
         types_or: [python]
         entry: |
           bash -c 'npm --prefix . run fix:ruff:format "${@}"' --
+
+      - id: dont-commit-detailed-processes
+        name: check that we don't commit processes_impacts.json
+        language: system
+        files: ^processes_impacts.json$
+        entry: |
+          bash -c 'exit 1'
```

Comment:
Nit: we may want to print some custom error message here, eg. "Prevented pushing detailed impacts data to public git repository. To update these, clone and push to the ecobalyse-private repository."